sdv/data_processing/anonymization.py

```diff
@@ -0,0 +1,89 @@
+"""Anonymization module for the ``DataProcessor``."""
+
+import inspect
+
+from faker import Faker
+from rdt.transformers import AnonymizedFaker
+
+SDTYPE_ANONYMIZERS = {
+    'phisical_address': {
+        'provider_name': 'address',
+        'function_name': 'address'
+    },
+    'email': {
+        'provider_name': 'internet',
+        'function_name': 'email'
+    },
+    'latitude': {
+        'provider_name': 'geo',
+        'function_name': 'latitude'
+    },
+    'longitude': {
+        'provider_name': 'geo',
+        'function_name': 'longitude'
+    },
+    'ipv4_address': {
+        'provider_name': 'internet',
+        'function_name': 'ipv4'
+    },
+    'ipv6_address': {
+        'provider_name': 'internet',
+        'function_name': 'ipv6'
+    },
+    'mac_address': {
+        'provider_name': 'internet',
+        'function_name': 'mac_address'
+    },
+    'name': {
+        'provider_name': 'person',
+        'function_name': 'name'
+    },
+    'phone_number': {
+        'provider_name': 'phone_number',
+        'function_name': 'phone_number'
+    },
+    'ssn': {
+        'provider_name': 'ssn',
+        'function_name': 'ssn'
+    },
+    'user_agent_string': {
+        'provider_name': 'user_agent',
+        'function_name': 'user_agent'
+    },
+}
+
+
+def _detect_provider_name(function_name):
+    function_name = getattr(Faker(), function_name)
```

Comment:
I think you're right, if we can move this to `RDT` at some point that would be great